.travis.yml

```diff
@@ -98,31 +98,30 @@ jobs:
         - skip
     - stage: conda-build
       install:
-        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-        - bash miniconda.sh -b -p $HOME/miniconda
-        - export PATH="$HOME/miniconda/bin:$PATH"
+        - cd ../
+        - bash ./tethys/scripts/install_tethys.sh --partial-tethys-install me -n tethys -s $PWD/tethys
+        - . ~/miniconda/etc/profile.d/conda.sh
+        - conda activate tethys
         - hash -r
         - conda config --set always_yes yes --set changeps1 no
         - conda update -q conda
         - conda install conda-build anaconda-client
-        - conda config --add channels bokeh/label/dev
-        - conda config --add channels conda-forge
-        - conda config --add channels tethysplatform/label/dev
-        - conda config --add channels tethysplatform
         - conda config --set anaconda_upload no
         - mkdir -p ~/conda-bld
       script:
         - echo "Building conda package"
         - export CONDA_BLD_PATH=~/conda-bld
-        - conda build ./conda.recipe
+        - conda build -c tethysplatform -c conda-forge ./tethys/conda.recipe
         - >
           if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
             echo "Uploading conda package";
             if [ -z "$TRAVIS_TAG" ]; then
               echo "No Branch tag is Present. Uploading to dev";
+              tethys gen metayaml --overwrite
```

Comment:
I missed this before, but it looks like the metayaml is being generated after the conda package has already been created, so it doesn't actually get used. We need to add the `conda build ...` statement after the `meta.yaml` file is generated.